packages/cxl-ui/scss/cxl-marketing-nav.scss

```diff
@@ -20,12 +20,30 @@
 #overlays-wrapper {
   position: fixed;
   display: flex;
-  border: 1px solid #d2d5da;
+  border: 1px solid var(--lumo-base-color);
+  // Use top and bottom border prevent panel divider from reaching full height.
+  border-top-width: var(--lumo-space-m);
+  border-bottom-width: var(--lumo-space-m);
   border-radius: 6px;
-  box-shadow: 0 8px 24px 0 rgba(0, 0, 0, 0.25);
+  // Second shadow set to emulate border
+  box-shadow: 0 8px 24px 0 rgba(0, 0, 0, 0.25), 0 0 1px 1px #D2D5DA;
+  opacity: 1;
+  transition: opacity 0.2s, left 0.1s;
```

Comment:
animations are not meant for aesthetics. They help hide an ugly flickering that happens because the menu is initally appended to the body and needs to be moved to the overlays-wrapper.